app/src/main/java/com/example/bigowlapp/activity/ScheduleReportActivity.java

```diff
@@ -59,7 +56,12 @@ private void subscribeToData() {
                 scheduleReportTitle.setText(schedule.getTitle());
                 scheduleReportStartTime.setText(schedule.getStartTime().toDate().toString());
                 scheduleReportEndTime.setText(schedule.getEndTime().toDate().toString());
-                scheduleReportLocation.setText(GeoLocationFormatter.formatLocation(this, schedule.getLocation()));
+
+                if (geoLocationFormatter == null) {
+                    geoLocationFormatter = new GeoLocationFormatter();
+                }
```

Comment:
move this into the OnStart() like  where the viewmodel if check is done. Should cause it to get some coverage